tests/test_buffer.py

```diff
@@ -728,17 +783,17 @@ def test_SimpleArrayPlex_constructor(self):
         for dtype in dtype_list_int:
             modmesh.SimpleArray((2, 3, 4), dtype=dtype, value=3)
             with self.assertRaisesRegex(
-                TypeError,
-                r"Data type mismatch, expected Python int"
+                    TypeError,
+                    r"Data type mismatch, expected Python int"
             ):
                 modmesh.SimpleArray((2, 3, 4), dtype=dtype, value=3.3)
 
         dtype_list_float = ["float32", "float64"]
         for dtype in dtype_list_float:
             modmesh.SimpleArray((2, 3, 4), dtype=dtype, value=3.0)
             with self.assertRaisesRegex(
-                TypeError,
-                r"Data type mismatch, expected Python float"
+                    TypeError,
```

Comment:
this format looks weird, 8 spaces?

Reply:
Yes, 2 levels of indentation.  It's required by [PEP-8](https://peps.python.org/pep-0008/#indentation).  I am fine either way as long as the linted code does not look horrible.

Reply:
I didn't know there a special rule for it, thanks for telling me.